src/distributions/beta.jl

```diff
@@ -1,16 +1,24 @@
 """
-Proof somewhere 
-parametrized in \\alpha and \\beta
+    Beta
+
+* Paramaterization
```

Comment:
```suggestion
* Parametrization
```

Reply:
This typo has been replicated in other parts, please ctrl+shift+F Paramaterization to correct all of them